modules/nw-multus-create-network-apply.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-create-network-apply_{context}"]
+= Creating an additional network attachment by applying a YAML manifest
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a YAML file with your additional network configuration, such as in the following example:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: next-net
+spec:
+  config: |-
+    {
+      "cniVersion": "0.3.1",
+      "name": "work-network",
+      "type": "host-device",
+      "device": "eth1",
+      "ipam": {
+        "type": "dhcp"
+      }
+    }
+----
+
+. To create the additional network, enter the following command:
++
+[source,terminal]
+----
+$ oc apply -f <file>.yaml
+----
++
+--
```

Comment:
Are the double dashes necessary? Interested in this for my own edification if nothing else, as removing them doesn't seem to change my preview and the OCP contribution guide gives this as an example:

```
----
code <variable_1>
code <variable_2>
----
+
where:

<variable_1>:: Specifies the explanation of the first variable.
<variable_2>:: Specifies the explanation of the first variable.
```

Reply:
@maxwelldb, interesting; I might have done this to ensure that subsequent numbers in the ordinal list were correct, so I've done it this way ever since. But I don't recall the specific case any longer. At worst, it's harmless here.